### DISCUSSION TITLE
DT in multiplayer

### DISCUSSION THREAD

**Discussion by halozy15** [ppy/osu#25346] [Ideas]
Is it possible to make DT more like free mod in lazer? it allows for players of high rank disparity to play in the same lobbies. After a DT player finishes map he can spectate the other players in multi

**Comment by Rekunan** [comment]
The main problem iirc is the lack of sync between players, and figuring out logistics with that (ex: the multiplayer leaderboard).

For now, they can use other mods (like DA).
